packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -313,6 +317,95 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   // stylelint-enable plugin/selector-bem-pattern
 }
 
+.mdc-textfield--with-leading-icon {
+  .mdc-textfield__leading-icon:not([tabindex]),
+  .mdc-textfield__leading-icon[tabindex="-1"] {
+    pointer-events: none;
+    cursor: default;
+  }
+
+  .mdc-textfield__trailing-icon:not([tabindex]),
```

Comment:
Shouldn't this be under `.mdc-textfield--with-trailing-icon` below instead? Or perhaps move both of these outside `.mdc-textfield--with-*-icon` altogether to avoid this mistake and reduce nesting?

If I remove the `tabindex="0"` from the trailing example in the demo page and do `$0.addEventListener('MDCTextfield:trailing-icon', () => console.log('event'))` on its root element, I still get `event` logged if I click the icon, which should not happen if this style is hooked up correctly.

Is it feasible for us to unit-test this, or would doing so require something like webdriver? Or should we perhaps actually hook up event handlers for these in the demo page to make it obvious, and add another checkbox below to remove the tabindexes?